docs/sources/enterprise/saml.md

```diff
@@ -109,6 +117,47 @@ Grafana provides configuration options that let you modify which keys to look at
 
 An example is `assertion_attribute_name = "givenName"` where Grafana looks within the assertion for an attribute with a friendly name or name of `givenName`. Both, the friendly name (e.g. `givenName`) or the name (e.g. `urn:oid:2.5.4.42`) can be used interchangeably as the value for the configuration option.
 
+### Configure team sync
+
+> Only available in Grafana v7.0+
+
+To be able to use Team Sync feature, set `assertion_attribute_groups` option to the attribute name where you store user groups. Then Grafana will use attribute values extracted from SAML assertion to add user into the groups with the same name configured on the External group sync tab.
+
+[Learn more about Team Sync]({{< relref "../enterprise/team-sync.md" >}})
+
+### Configure role sync
+
+> Only available in Grafana v7.0+
+
+Role sync allows to map user roles from identity provider to Grafana. To enable role sync, set `assertion_attribute_role` option to the attribute name where the role information will be extracted from. After that, configure desired attribute values for each of Grafana roles: Editor, Admin and Super Admin. If obtained role doesn't match any of configured values, `Viewer` role will be assigned to the user. Example configuration:
+
+```bash
+[auth.saml]
+assertion_attribute_role = role
+role_values_editor = editor, developer
+role_values_admin = admin, operator
+role_values_grafana_admin = superadmin
+```
+
+Refer to [Organization roles]({{< relref "../permissions/organization_roles.md" >}}) for more information about roles and permissions in Grafana.
+
+**Important**: when role sync is configured, any changes of user roles and organization membership made in Grafana will be overwritten on next user login. Assign user organizations and roles in the IdP instead.
+
+### Configure organization mapping
+
+> Only available in Grafana v7.0+
+
+Organization mapping allows you to assign users to particular organization in Grafana depending on configured `assertion_attribute_org` attribute value obtained from identity provider. Set `assertion_attribute_org` to the attribute name you store organization info in. Then configure `org_mapping` option to map organization from IdP to Grafana organization specified by id. For example, `org_mapping = Engineering:2, Sales:3` assigns users from `Engineering` organization to the Grafana organization with id `2` and users from `Sales` - to the org with id `3`. You can specify multiple organizations both for the IdP and Grafana:
+
```

Comment:
I think we should show setting `assertion_attribute_org` here too to clearly show that it's also needed.

Reply:
@xlson , does this look good to you?

Reply:
I think that looks great, a lot easier to follow now.